pyproject.toml

```diff
@@ -1,19 +1,19 @@
 [project]
 name = "chronos"
-version = "1.2.1"
+version = "1.3.0"
 requires-python = ">=3.8"
 license = { file = "LICENSE" }
 dependencies = [
     "torch~=2.0",         # package was tested on 2.2
     "transformers~=4.30",
-    "accelerate",
+    "accelerate~=0.32",
 ]
 
 [project.optional-dependencies]
 test = ["pytest~=8.0", "numpy~=1.21"]
 typecheck = ["mypy~=1.9"]
-training = ["gluonts[pro]", "numpy", "tensorboard", "typer", "typer-config", "joblib", "scikit-learn"]
-evaluation = ["gluonts[pro]", "datasets", "numpy", "typer"]
+training = ["gluonts[pro]~=0.15", "numpy~=1.21", "datasets~=2.18", "typer", "typer-config", "joblib", "scikit-learn", "tensorboard"]
+evaluation = ["gluonts[pro]~=0.15", "numpy~=1.21", "datasets~=2.18", "typer"]
```

Comment:
We could also use numpy 2 with gluonts 0.16. Changes compared to 0.15 were rather minimal https://github.com/awslabs/gluonts/releases/tag/v0.16.0. But we can also do this in two steps and have a follow-up issue/PR

Reply:
Let's do this in a follow-up then?